luminex/resources/schemas/dbscripts/postgresql/luminex-25.000-25.001.sql

```diff
@@ -0,0 +1,17 @@
+-- This index overlaps with pk_luminex_runexclusion
+DROP INDEX luminex.idx_luminexrunexclusion_runid;
+-- This index overlaps with uq_analyte_lsid
+DROP INDEX luminex.ix_luminexdatarow_lsid;
```

Comment:
`uq_analyte_lsid` is on a different table. `luminex.analyte` vs `luminex.datarow`. Am I missing something?